### PR TITLE
fix: Ingress Controller Configuration

### PR DESCRIPTION
**Pull request by manish-jangra**
This Pull Request will limit the Ingress Controler Configuration to  `stone-stg-host` && `stone-prd-host1`  clusters only

### CODE REVIEW

**Review by hugares — APPROVED** [review]
/lgtm

File: argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml [hunk 1]
```diff
@@ -10,17 +10,22 @@ spec:
         generators:
           - clusters:
               values:
-                environment: ""
-                clusterDir: ""
+                sourceRoot: configs/ingresscontroller
+                environment: staging
+                clusterDir: base
```
Comment (hugares) [inline argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml:15]: By defaulting all `clusters to configs/ingresscontroller/base`, this means that the time out patch will be applied to all clusters, not only dev, stg-host and prd-host1. Is this what you want?

Comment (hugares) [reply argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml:15]: ignore my comment, we detault to configs/ingresscontroller/staging/base which does not exists. I think we are good

File: argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml [hunk 2]
```diff
@@ -10,17 +10,22 @@ spec:
         generators:
           - clusters:
               values:
-                environment: ""
-                clusterDir: ""
+                sourceRoot: configs/ingresscontroller
+                environment: staging
```
Comment (hugares) [inline argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml:14]: we are missing somthing that will replace environment value from staging to production in the production overlay

Comment (manish-jangra) [reply argo-cd-apps/base/host/ingresscontroller/ingresscontroller.yaml:14]: should be fixed now, added a patch for this in its overlay.

**Pull request merged**